extension/chrome/settings/setup/setup-render.ts

```diff
@@ -98,8 +98,8 @@ export class SetupRenderModule {
     } catch (e) {
       return await Settings.promptToRetry(e, Lang.setup.failedToCheckIfAcctUsesEncryption, () => this.renderSetupDialog());
     }
-    if (keyserverRes.pubkey) {
-      const pub = await KeyUtil.parse(keyserverRes.pubkey);
+    if (keyserverRes.pubkeys.length) {
+      const pub = await KeyUtil.parse(keyserverRes.pubkeys[0]); // todo: ?
       this.view.acctEmailAttesterPubId = pub.id;
```

Comment:
Yup - please make `acctEmailAttesterPubId` an array. Then somewhere here will be `acctEmailAttesterPubId !== prv.id` or something similar, to see if the public key is consistent. Please change that to check if `acctEmailAttesterPubId.includes(prv.id)`.